### PR TITLE
doc(support-policy) update to new support policy

### PR DESCRIPTION
**Pull request by GayleNeumann**
-- Updated content to new Support Policy, per Shane's doc: https://docs.google.com/document/d/1mCe3gX-3r3QaSLLMhj_vZU1Bn-Yu6SUVXQPwAbvhFp4/edit#
-- This update to be released on Monday, Nov 30, 2020, per Tom in Legal. 
-- Coordinate this doc with Shane/Product, Jimmy/Customer Support, and Tom/Legal.

See https://deploy-preview-2476--kongdocs.netlify.app/enterprise/2.2.x/support-policy/

### CODE REVIEW

File: app/enterprise/2.2.x/support-policy.md [hunk 1]
```diff
@@ -3,37 +3,73 @@ title: Kong Enterprise Version Support
...
 
-* Access to all error corrections, bug fixes, software updates, and software upgrades to the Software that Kong makes generally available to other Kong enterprise customers who have purchased a license to the same Software purchased by Customer. 
+* The **Latest Major/Minor Version Release** is the most recent chronologically generally available Minor Version release of a given Major Version.
 
-* Development of workarounds or patches for customers upon request, and where technically possible, for the following: Urgent issues; Critical and High Security Vulnerabilities; High priority issues where the product in a non-production environment is nearing a critical milestone, is unusable, and no workaround is available.
+For customers with an active enterprise support agreement, beginning with Kong Enterprise release version 2.1.3.x<sup>*</sup>. Kong provides support for each Major Version release of Kong Enterprise for 24 months from the date of the First Major Version Release.
```
Comment (eskibars) [inline app/enterprise/2.2.x/support-policy.md:19]: The last sentence here should read "Kong provides support for each Major Version release of Kong Enterprise for 24 months from the release date of the Minor Version."

File: app/enterprise/2.2.x/support-policy.md [hunk 2]
```diff
@@ -3,37 +3,73 @@ title: Kong Enterprise Version Support
...
+After the product hits the end of the support period, Kong will provide limited support to help the customer upgrade to a fully supported version of Kong Enterprise for up to an additional 12 month sunset period. Kong will not provide patches for software covered by this sunset period. If there is an issue that requires a patch during this period, the customer will need to upgrade to a newer Kong Enterprise version covered by 24 month active support.
 
-> Table 1: End of Support Date
+## Bug Fix Guidelines
+Unfortunately, all software is susceptible to bugs. Kong seeks to remedy bugs through a structured protocol as follows:
 
-End of life support will be limited to technical assistance only, with no source code patches or new patch level releases. Kong's support team may provide recommendations to the customer, but no software patches will be provided. If there is an issue that requires a patch, customers will need to upgrade to a newer version covered by the full Support Policy.
+* Serious security vulnerabilities are treated with the utmost priority. See here for our security vulnerability reporting and remedy process, including how to report a vulnerability.
```
Comment (eskibars) [inline app/enterprise/2.2.x/support-policy.md:36]: "here" should have a link to /VERSION/kong-security-update-process/

**Review by eskibars — CHANGES_REQUESTED** [review]
Left 2 small comments/revision requests.  Other than that LGTM

**Comment by eskibars** [comment]
Oh, 1 more request, the bottom says `The above is a summary only and is qualified by Kong’s Support and Maintenance Policy.`.  Let's link that `Support and Maintenance Policy` to https://konghq.com/supportandmaintenancepolicy/

File: app/enterprise/2.2.x/support-policy.md [hunk 3]
```diff
@@ -3,37 +3,73 @@ title: Kong Enterprise Version Support
 toc: false
```
Comment (lena-larionova) [inline app/enterprise/2.2.x/support-policy.md:3]:
```suggestion
```

**Review by lena-larionova — APPROVED** [review]
LGTM

**Comment by eskibars** [comment]
Thanks @GayleNeumann this LGTM!

**Review by eskibars — APPROVED** [review]

**Pull request merged**